lib/Alchemy/Phrasea/Twig/Camelize.php

```diff
@@ -11,21 +11,11 @@
 
 namespace Alchemy\Phrasea\Twig;
 
-use Alchemy\Phrasea\Utilities\String\Camelizer;
+use Alchemy\Phrasea\Utilities\StringHelper;
 
 class Camelize extends \Twig_Extension
```

Comment:
There are reasons no more to call this extension Camelize. StringHelper instead?